QEfficient/exporter/export_utils.py

```diff
@@ -43,7 +43,7 @@ def export_onnx(
     # Inspect the model's forward method arguments
     pt_model_code = pt_model.forward.__code__
     pt_input_names = pt_model_code.co_varnames[1 : pt_model_code.co_argcount]
-
+    starcoder1_inputs = False
```

Comment:
Move this to line 71, just above whether it's true or false is determined. Also, rename this variable to `kv_cache_3d`.

Reply:
You could make it a one liner as `kv_cache_3d = len(inputs["past_key_values"][0][0].shape) == 3`